### DISCUSSION TITLE
PyUSB 1.3.0 has been postponed

### DISCUSSION THREAD

**Discussion by jonasmalacofilho** [pyusb/pyusb#400] [Announcements]
Hi all!

Since there have been no significant changes to PyUSB since 1.2.1,[^1] I have cancelled the release previously scheduled for January 3rd.  This follows our [documented development process](https://github.com/pyusb/pyusb/blob/master/docs/internal/process.md), and takes into account that we were already in the pre-release freeze.

PyUSB 1.3.0 has been tentatively rescheduled to July 2022.  In the meantime, 1.2.x may receive additional patch releases as necessary.

Jonas

[^1]: https://github.com/pyusb/pyusb/compare/v1.2.1...043babbe910cfc191185864dbc694513535f094f

**Comment by ve2dtk** [comment]
Where I can get the PyUSB 1.2.1 so I can use it with my raspberry PI to control the Sainsmart 8 USB Channel relay based on the  FTDI FT245R.

Thank you very much

**Reply by mcuee** [reply]
https://github.com/pyusb/pyusb/releases

pyusb 1.3.1 has been released.
